src/main/java/millom/sandbox/mars/weather/service/JdkService.java

```diff
@@ -0,0 +1,34 @@
+package millom.sandbox.mars.weather.service;
+
+import jakarta.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+import millom.sandbox.mars.weather.CustomException.JdkException;
```

Comment:
I think it would be better to move the JDK API specific code into its own namespace. F.ex. to millom.sandbox.jdk.<something>.

Reply:
`import millom.sandbox.jdk.CustomException.JdkException;
`